packages/phone-number-privacy/signer/azure-templates/container-parameters.json

```diff
@@ -90,6 +98,9 @@
         },
         "logAnalyticsWorkspaceKey": {
             "value": "TODO"
-        }
+        },
+        "userAssignedIdentity": {
```

Comment:
Thanks!! Can you also update the `azure-templates/README.md` with the prerequisite step to deploy the UserAssignedIdentity?

Reply:
done!